reinforcement_learning/rlclientlib/schema/OutcomeEvent.fbs

```diff
@@ -0,0 +1,12 @@
+﻿// EventHubInteraction Schema used by FlatBuffer
+
+namespace VW.Events;
+
+table OutcomeEvent {
+    EventId:string;
+    v:string;
+}
+
+table OutcomeEventBatch {
+	Events:[OutcomeEvent];
```

Comment:
>	 [](start = 0, length = 1)

nit: inconsistent tabs vs spaces in this file.

Reply:
yes. Will fix this.

---
In reply to: [228273114](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1653#discussion_r228273114) [](ancestors = 228273114)